int32 bbox_xmin
int32 bbox_ymin
int32 bbox_xmax
int32 bbox_ymax
string[] label
float32[] score
string max_label
float32 max_score